flink-libraries/flink-table/src/main/scala/org/apache/flink/table/calcite/CalciteConfig.scala

```diff
@@ -81,6 +84,25 @@ class CalciteConfigBuilder {
   }
 
   /**
+    * Replaces the built-in decoration rule set with the given rule set.
```

Comment:
Can you add a more detailed description what decoration rules, norm rules etc. are. You can reuse your PR text: "after the volcano optimization, Decoration phase is dedicated for rewriting predicated logical plan and is independent of cost module"